implants/golem/src/main.rs

```diff
@@ -68,6 +82,15 @@ fn main() -> anyhow::Result<()> {
                 .multiple_occurrences(false)
                 .required(false),
         )
+        .arg(
```

Comment:
Should probably be an environment variable too, may not always want to pass the password in CLI.